indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -161,6 +175,27 @@ public String apply(Instance input)
 
       log.info("Created instances: %s", instanceIds);
 
+      if (envConfig.getNodeData().getTags() != null) {
+        final List<Tag> tags = envConfig
+            .getNodeData()
+            .getTags()
+            .entrySet()
+            .stream()
```

Comment:
We're still on java7. See https://groups.google.com/d/msg/druid-development/6I9uhO3ZIeQ/WG-K6JdEEQAJ